pegasus/router.rb

```diff
@@ -316,7 +316,7 @@ def self.load_configs_in(dir)
   not_found do
     status 404
     path = resolve_template('views', settings.template_extnames, '/404')
-    document path
+    (document path).tap{dont_cache}
```

Comment:
The return value of the `not_found` block is the response body rendered by the server, and `document path` internally calls `cache_control` to set the default expiration for document pages to 1 hour. So we could get the right behavior with `(document path).tap{dont_cache}` or something like
```
page = document path
dont_cache
page
```

I thought the `tap` version would be more concise but could change to the latter if you think it's more readable